daemon/logger/amqp/amqp.go

```diff
@@ -0,0 +1,419 @@
+// +build linux
```

Comment:
So far I've only tested it on Linux. Once I've made the major changes to the current code I'll look at adding support for Windows.